pkg/cmd/workspace/delete.go

```diff
@@ -68,3 +89,30 @@ var DeleteCmd = &cobra.Command{
 		util.RenderInfoMessage(fmt.Sprintf("Workspace %s successfully deleted", *workspace.Name))
 	},
 }
+
+func init() {
+	DeleteCmd.Flags().BoolVarP(&allFlag, "all", "a", false, "Delete all workspaces")
+	DeleteCmd.Flags().BoolVarP(&yesFlag, "yes", "y", false, "Confirm deletion without prompt")
+}
+
+func deleteAllWorkspaces() {
```

Comment:
To be consistent with our code style, functions should not exit the process themselves but return an error that the calling function will handle.

In this case, `deleteAllWorkspaces` would return an error and then in the cobra command you would handle the error with `log.Fatal(err)`